GEOSldas_App/util/postproc/ObsFcstAna_stats/helper/plot.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from mpl_toolkits import basemap
+
+def plotMap(
```

Comment:
This is extremely generic and should be moved to ./util/shared/python if possible.